src/Microsoft.Diagnostics.Runtime/src/Implementation/ClrmdHeap.cs

```diff
@@ -173,11 +171,12 @@ private IReadOnlyList<ClrSegment> Initialize()
         {
             lock (_sync)
             {
-                _helpers.CreateSegments(this,
-                                        out IReadOnlyList<ClrSegment> segments,
-                                        out IReadOnlyList<AllocationContext> allocContext,
-                                        out IReadOnlyList<FinalizerQueueSegment> fqRoots,
-                                        out IReadOnlyList<FinalizerQueueSegment> fqObjects);
+                _helpers.CreateSegments(
```

Comment:
I dislike this refactor, is there a way to turn this one off?

Reply:
Aligning by opening parentheses will result in inconsistencies...

Reply:
I suppose that's fine, but it's going to be a hard habit to break after 20 years of doing it the other way.  =)

Reply:
Also, is it a habit to spell out .NET as .Net? It should be all capitalized.

[.NET Glossary](https://github.com/dotnet/docs/blob/master/docs/standard/glossary.md#net)

[Microsoft Style Guide](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/a-z-word-list-term-collections/n/net.md)